server/src/test/java/org/elasticsearch/cluster/coordination/Zen1IT.java

```diff
@@ -71,12 +71,10 @@
 public class Zen1IT extends ESIntegTestCase {
 
     private static Settings ZEN1_SETTINGS = Coordinator.addZen1Attribute(true, Settings.builder()
-        .put(TestZenDiscovery.USE_ZEN2.getKey(), false)
-        .put(TestZenDiscovery.USE_MOCK_PINGS.getKey(), false)) // Zen2 does not know about mock pings
-        .build();
+        .put(DiscoveryModule.DISCOVERY_TYPE_SETTING.getKey(), DiscoveryModule.ZEN_DISCOVERY_TYPE)).build();
 
     private static Settings ZEN2_SETTINGS = Settings.builder()
```

Comment:
should we rename the "zen2" discovery type to "default"? This means no need for the "zen" name anymore